networkconfig/config.go

```diff
@@ -75,3 +75,14 @@ func (n NetworkConfig) SlotsPerEpoch() uint64 {
 func (n NetworkConfig) GetGenesisTime() time.Time {
 	return time.Unix(int64(n.Beacon.MinGenesisTime()), 0)
 }
+
+// Domain returns current domain type of the network
+func (n NetworkConfig) DomainType() spectypes.DomainType {
+	if n.AlanForked(n.Beacon.EstimatedCurrentSlot()) {
+		forkDomain := make([]byte, 4)
+		copy(forkDomain, n.domainType[:2])
+		copy(forkDomain[2:], []byte{0x1})
+		return spectypes.DomainType(forkDomain)
+	}
+	return n.domainType
+}
```

Comment:
@moshe-blox changed to this. So after we reach Alan fork 4th byte will be changed to 0x1

Reply:
@moshe-blox @MatusKysel does it make sense to branch out from `alan/no-fork` and redo the changes? will be much cleaner at new PR?

Reply:
Ok closing, reopening at new PR